### PR TITLE
feat: accessor for filtering 3D Asset Mappings [BND3D-676]

### PR DESCRIPTION
**Pull request by larsmoa**
This enables accessing endpoint https://docs.cognite.com/api/v1/#operation/filter3DAssetMappings from the SDK.

### CODE REVIEW

**Comment by polomani** [comment]
@larsmoa wouldn't it be more efficient to run tests locally first?

**Comment by larsmoa** [comment]
@polomani: local-testservice broken for me and not something I wanted to dive into. In retrospect; maybe.

**Comment by haakonflatval-cognite** [comment]
@polomani Are there any instructions anywhere as to how to run the tests locally?

File: packages/stable/src/api/3d/assetMappings3DApi.ts [hunk 1]
```diff
@@ -8,9 +8,17 @@ import {
...
   DeleteAssetMapping3D,
+  Filter3DAssetMappingsQuery,
 } from '../../types';
 
 export class AssetMappings3DAPI extends BaseResourceAPI<AssetMapping3D> {
+  /**
+   * @override
+   */
+  protected get searchUrl() {
+    return this.url('filter');
```
Comment (polomani) [inline packages/stable/src/api/3d/assetMappings3DApi.ts]: this shouldn't be needed

File: packages/stable/src/api/3d/assetMappings3DApi.ts [hunk 2]
```diff
@@ -30,6 +38,30 @@ export class AssetMappings3DAPI extends BaseResourceAPI<AssetMapping3D> {
...
+    modelId: CogniteInternalId,
+    revisionId: CogniteInternalId,
+    query: Filter3DAssetMappingsQuery
+  ): CursorAndAsyncIterator<AssetMapping3D> => {
+    const path = `${this.encodeUrl(modelId, revisionId)}/list`;
+    return super.listEndpoint(
+      params =>
+        this.post<CursorResponse<AssetMapping3D[]>>(path, { data: params }),
+      query
+    );
```
Comment (polomani) [inline packages/stable/src/api/3d/assetMappings3DApi.ts:55]: lines 58-62 above should be possible to replace with
```
super.listEndpoint(this.callListEndpointWithPost, query);
```

Comment (haakonflatval-cognite) [reply packages/stable/src/api/3d/assetMappings3DApi.ts:55]: Would that call use the right URL? The URL must contain `modelId` and `revisionId`, which must be communicated somehow.

Comment (polomani) [reply packages/stable/src/api/3d/assetMappings3DApi.ts:55]: yes, you're right. then disregard my previous comment

**Review by vegardok — APPROVED** [review]

**Pull request merged**